docker/compose/docker-compose.sqlite-tika.arm.yml

```diff
@@ -0,0 +1,84 @@
+# docker-compose file for running paperless from the docker container registry.
+# This file contains everything paperless needs to run.
+# Paperless supports amd64, arm and arm64 hardware.
+#
+# All compose files of paperless configure paperless in the following way:
+#
+# - Paperless is (re)started on system boot, if it was running before shutdown.
+# - Docker volumes for storing data are managed by Docker.
+# - Folders for importing and exporting files are created in the same directory
+#   as this file and mounted to the correct folders inside the container.
+# - Paperless listens on port 8000.
+#
+# SQLite is used as the database. The SQLite file is stored in the data volume.
+#
+# iwishiwasaneagle/apache-tika-arm docker image is used to enable arm64 arch
+# which apache/tika does not currently support.
+#
+# In addition to that, this docker-compose file adds the following optional
+# configurations:
+#
+# - Apache Tika and Gotenberg servers are started with paperless and paperless
+#   is configured to use these services. These provide support for consuming
+#   Office documents (Word, Excel, Power Point and their LibreOffice counter-
+#   parts.
+#
+# To install and update paperless with this file, do the following:
+#
+# - Copy this file as 'docker-compose.yml' and the files 'docker-compose.env'
+#   and '.env' into a folder.
+# - Run 'docker-compose pull'.
+# - Run 'docker-compose run --rm webserver createsuperuser' to create a user.
+# - Run 'docker-compose up -d'.
+#
+# For more extensive installation and update instructions, refer to the
+# documentation.
+
+version: "3.4"
+services:
+  broker:
+    image: redis:6.0
+    restart: unless-stopped
+    volumes:
+      - redisdata:/data
+
+  webserver:
+    image: ghcr.io/paperless-ngx/paperless-ngx:latest
+    restart: unless-stopped
+    depends_on:
+      - broker
+      - gotenberg
+      - tika
+    ports:
+      - 8000:8000
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8000"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+    volumes:
+      - data:/usr/src/paperless/data
+      - media:/usr/src/paperless/media
+      - ./export:/usr/src/paperless/export
+      - ./consume:/usr/src/paperless/consume
+    env_file: docker-compose.env
+    environment:
+      PAPERLESS_REDIS: redis://broker:6379
+      PAPERLESS_TIKA_ENABLED: 1
+      PAPERLESS_TIKA_GOTENBERG_ENDPOINT: http://gotenberg:3000
+      PAPERLESS_TIKA_ENDPOINT: http://tika:9998
+
+  gotenberg:
+    image: thecodingmachine/gotenberg
+    restart: unless-stopped
+    environment:
+      DISABLE_GOOGLE_CHROME: 1
+
+  tika:
+    image: iwishiwasaneagle/apache-tika-arm@sha256:c1b16bcadcb97a08d47a09fc99bea4d47b63cec06bbf4f437b99c2c3b17181eb
```

Comment:
```suggestion
    image: iwishiwasaneagle/apache-tika-arm@sha256:881208a3a6d25418a67dddbbc44d1c73528f9bfc2538b5207ceff60a86a8e11f
```

Reply:
Thanks for building it! Although this is the sha of the manifest I think we need to use:

```sha256:a78c25ffe57ecb1a194b2859d42a61af46e9e845191512b8f1a4bf90578ffdfd```

Reply:
This made me realize that pinning a SHA would pin the architecture. I'm not aware of a method of pinning a specific build without pinning the SHA though... Any suggestions? 

--- 

![image](https://user-images.githubusercontent.com/12383029/159691796-0c0fb80f-1a5e-4018-882e-e9e2979a620e.png)

2.1.0-full:
 - amd64 -> 881208a3a6d25418a67dddbbc44d1c73528f9bfc2538b5207ceff60a86a8e11f
 - arm64 -> 4aed0fc5611d6e40816537216bc1222e8bb45d20414afd768592b241da93274f
 - armv7 -> 301609fb6e76dc11eb515e93b1664939577f02e57c6f5767ceb082419fdf676f

Reply:
Right, the way around this IIRC is to pin the sha of the manifest for those three archs. That would be the `a78c25f...`. This is normally found when doing a `docker manifest inspect` but I didn't see it there (yay, experimental features). 

Instead I ran [regctl](https://github.com/regclient/regclient)

```bash
$ ./regctl manifest get iwishiwasaneagle/apache-tika-arm:2.1.0-full
Name:        iwishiwasaneagle/apache-tika-arm:2.1.0-full
MediaType:   application/vnd.docker.distribution.manifest.list.v2+json
Digest:      sha256:a78c25ffe57ecb1a194b2859d42a61af46e9e845191512b8f1a4bf90578ffdfd

Manifests:

  Name:      docker.io/iwishiwasaneagle/apache-tika-arm:2.1.0-full@sha256:881208a3a6d25418a67dddbbc44d1c73528f9bfc2538b5207ceff60a86a8e11f
  Digest:    sha256:881208a3a6d25418a67dddbbc44d1c73528f9bfc2538b5207ceff60a86a8e11f
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64

  Name:      docker.io/iwishiwasaneagle/apache-tika-arm:2.1.0-full@sha256:4aed0fc5611d6e40816537216bc1222e8bb45d20414afd768592b241da93274f
  Digest:    sha256:4aed0fc5611d6e40816537216bc1222e8bb45d20414afd768592b241da93274f
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64

  Name:      docker.io/iwishiwasaneagle/apache-tika-arm:2.1.0-full@sha256:301609fb6e76dc11eb515e93b1664939577f02e57c6f5767ceb082419fdf676f
  Digest:    sha256:301609fb6e76dc11eb515e93b1664939577f02e57c6f5767ceb082419fdf676f
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm/v7
```

You can confirm this works by running `docker pull iwishiwasaneagle/apache-tika-arm:2.1.0-full@sha256:a78c25ffe57ecb1a194b2859d42a61af46e9e845191512b8f1a4bf90578ffdfd`

Reply:
Can confirm this works

```bash
docker manifest inspect iwishiwasaneagle/apache-tika-arm@sha256:a78c25ffe57ecb1a194b2859d42a61af46e9e845191512b8f1a4bf90578ffdfd
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1796,
         "digest": "sha256:881208a3a6d25418a67dddbbc44d1c73528f9bfc2538b5207ceff60a86a8e11f",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1796,
         "digest": "sha256:4aed0fc5611d6e40816537216bc1222e8bb45d20414afd768592b241da93274f",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1796,
         "digest": "sha256:301609fb6e76dc11eb515e93b1664939577f02e57c6f5767ceb082419fdf676f",
         "platform": {
            "architecture": "arm",
            "os": "linux",
            "variant": "v7"
         }
      }
   ]
}
```

Reply:
That's the suggestion updated if you want to double check the sha https://github.com/paperless-ngx/paperless-ngx/pull/479#discussion_r833122430

Reply:
Committed just now. Thank you for maintaining this image!